examples/grid_example/public/utils.ts

```diff
@@ -45,23 +60,36 @@ export const dashboardInputToGridLayout = ({
 }): GridLayoutData => {
   const layout: GridLayoutData = {};
   Object.values(rows).forEach((row) => {
-    const { collapsed, ...rest } = row;
+    const { collapsed, y, ...rest } = row;
     layout[row.id] = {
       ...rest,
+      type: 'section',
+      row: y,
       panels: {},
       isCollapsed: collapsed,
     };
   });
 
   Object.keys(panels).forEach((panelId) => {
     const gridData = panels[panelId].gridData;
-    layout[gridData.row ?? 'first'].panels[panelId] = {
-      id: panelId,
-      row: gridData.y,
-      column: gridData.x,
-      width: gridData.w,
-      height: gridData.h,
-    };
+    if (gridData.section) {
+      (layout[gridData.section] as GridSectionData).panels[panelId] = {
+        id: panelId,
+        row: gridData.y,
+        column: gridData.x,
+        width: gridData.w,
+        height: gridData.h,
+      };
+    } else {
+      layout[panelId] = {
+        id: panelId,
+        type: 'panel',
+        row: gridData.y,
+        column: gridData.x,
+        width: gridData.w,
+        height: gridData.h,
+      };
+    }
```

Comment:
nit:

```suggestion
 const panelData = {
      id: panelId,
      row: gridData.y,
      column: gridData.x,
      width: gridData.w,
      height: gridData.h,
    };
    if (gridData.section) {
      (layout[gridData.section] as GridSectionData).panels[panelId] = panelData;
    } else {
      layout[panelId] = { type: 'panel', ...panelData };
    }
```

Reply:
Done in dbdfa8f878b9ef2628c98a85f3aea0b62a429d41